src/ontology/cl-edit.owl

```diff
@@ -29524,6 +29525,16 @@ AnnotationAssertion(rdfs:label obo:CL_4028006 "alveolar type 2 fibroblast cell")
 SubClassOf(obo:CL_4028006 obo:CL_0002241)
 SubClassOf(obo:CL_4028006 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002299))
 
+# Class: obo:CL_4030000 (choroidal melanocyte)
+
+AnnotationAssertion(<http://geneontology.org/formats/oboInOwl#created_by> obo:CL_4030000 "http://orcid.org/0000-0002-1773-2692")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31819095"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:34082079"^^xsd:string) obo:IAO_0000115 obo:CL_4030000 "Choroidal melanocytes are melanin-producing cells in the uvea of the human eye. Their cellular functions and extracellular interactions are distinct to skin melanocytes. They are involved in photoprotection, regulation of oxidative damage and immune responses. Choroidal melanocytes might have both neuroectodermal as well as mesectodermal origins.")
```

Comment:
I think convention is to have "A [parent class] that..." and not to use the name in the first sentence (definiendum). So: "A melanocyte in the uvea of the human eye" (or should it be choroid?) - also is this human specific? 

Chris Mungall wrote a really great blog that might help (it is quite a bit of a read though) - https://douroucouli.wordpress.com/2019/07/08/ontotip-write-simple-concise-clear-operational-textual-definitions/

Reply:
I think "Their cellular functions and extracellular interactions are distinct to skin melanocytes." and "Choroidal melanocytes might have both neuroectodermal as well as mesectodermal origins." should be in rdfs:comments instead

Reply:
Thanks for the feedback, @shawntanzk. I used the definition provided by the requestor but can break it down per your suggestions. The references in the ticket both reference humans, but can do additional searching for non-human references.

Reply:
I found references to this cell type in mice, cats and dogs, so I removed the reference to humans.

Reply:
@shawntanzk, valid point on uvea vs choroid location. 
PMID: 31819095 states "Choroidal melanocytes (HCMs) are melanin-producing cells in the vascular uvea of the human eye (iris, ciliary body and choroid)". 
The requestor assigned optic choroid as the anatomical location and the name itself does suggest it is part of the choroid. In these cases does it warrant a follow up with the requestor or should we decide based on our own research?

Reply:
from personal experience, I try to do what I can from research and then in the end check with requester or an expert if it is alright/what needs changing - tends to get things done a lot faster that way (so I basically write everything up how I think is right then ask) But perhaps more importantly, the textual definition should match up with your logical definition as much as possible.

Personally, I would use vascular uvea to be safe but also change the logical axiom as such (if you put choroid, then if cannot be in iris and ciliary body, but if you put vascular uvea, it is still in choroid) - really annoying given the name of the cell though lol. But probably also check with someone more experienced than me, maybe @paolaroncaglia could comment on how to handle this?

Reply:
Thank you for the guidance, @shawntanzk. The requester agreed to align the definition with the anatomical structure in EA, so both are set to 'uvea'.

Reply:
I think it would be useful to have a has_exact_synonym as 'uvea melanocyte' in this case - given the equivalent class axiom - sorry for being a pain >.<

Reply:
maybe "uveal"

Reply:
lol, yes, that >.<

Reply:
Not a pain, @shawntanzk. I appreciate the feedback and added the synonym.